packages/generate-pdf/src/index.ts

```diff
@@ -135,18 +136,18 @@ async function getPdfMakeContentFromObject(
   extraTranslations: ExtraTranslations = null,
 ): Promise<TDocumentDefinitions['content']> {
   const refSchemaUrl = new URL(certificate.RefSchemaUrl);
-  const schemaConfig = getSchemaConfig(refSchemaUrl);
+  const schemaConfig: SchemaConfig = getSchemaConfig(refSchemaUrl);
   const certificateLanguages = getCertificateLanguages(certificate);
   if (!translations) {
     translations = certificateLanguages?.length ? await getTranslations(certificateLanguages, schemaConfig) : {};
   }
 
-  const { type } = castCertificate(certificate);
-
-  const externalStandards: ExternalStandards[] =
-    schemaToExternalStandardsMap[type]
-      .map((schemaType) => get(certificate, schemaType, undefined))
-      .filter((externalStandards) => externalStandards) || [];
+  const type = getCertificateType(schemaConfig);
```

Comment:
Could you add a comment above `generatePdf` with the JSDoc syntax saying that the certificate passed should be validated before hand.